lib/pdk/cli/test/unit.rb

```diff
@@ -20,20 +20,20 @@ module PDK::CLI
       report = nil
 
       if opts[:list]
-        puts _('List of all available unit tests: (TODO)')
+        PDK::Test::Unit.list
```

Comment:
I would recommend moving the output logic from `PDK::Test::Unit.list` and into here. If it just returns an array of test names and leaves it to the CLI to print them out for the user, it can be potentially be utilised in other places in the future (e.g. validating the contents of `opts[:tests]` before handing off to rspec).